Representable/representable
# Select To View State Map

**changes**
- Adds select to view functionality on state map page
- Adds ability to toggle back to viewing all communities at once

**to test**
- Go to state map page
- Select one or more of the checkboxes and u should only see that community(ies).
- Click the "Show All" button link and all the checkboxes should be unchecked and all the communities should be shown.
